spaces/farm/skin.scss

```diff
@@ -0,0 +1,9 @@
+.name {
```

Comment:
This seem to be the default skin, if you haven't changed this file you can remove it.

Reply:
sure, would I be able to modify it in the future?